### DISCUSSION TITLE
security fix question

### DISCUSSION THREAD

**Discussion by klssstis** [openssl/openssl#21787] [Q&A]
Recently you have made the following corrections: https://github.com/openssl/openssl/commit/0577dbad
https://github.com/openssl/openssl/commit/e16c0103.
I think it's memory leak problem's 
do you plan to run CVE or do you have an internal tracker for these bugs

**Comment by kroeckx** [comment]
We will most likely not issue a CVE for them, since we don't see how an attacker can trigger it.

**Comment by klssstis** [comment]
thanks for the answer)

**Comment by mattcaswell** [comment]
Also, those issues only affect the master branch.  They do not affect any stable release. We only issue CVEs for stable releases.

**Comment by klssstis** [comment]
Good afternoon.
sorry if my question sounds stupid, but doesn't the master branch code correlate with the code in the release product?

**Comment by mattcaswell** [comment]
The master branch is what will become 3.2 when it is released.

**Comment by klssstis** [comment]
Yes, I understand that.
but by the same logic, version 3.1.2 is now in the master branch.
for example, the code section that was fixed by commit https://github.com/openssl/openssl/commit/e16c0103 did not change 2 months before the fix (lines 479-739), which means it should have been included in 3.1.2 on August 1.

**Reply by mattcaswell** [reply]
> but by the same logic, version 3.1.2 is now in the master branch.

No. The 3.1.2 version is part of the openssl-3.1 branch. Those two branches diverged two years ago:

````
$ git merge-base master openssl-3.1
59f4a51a7f2c53b9fd161b032d0fcb8a85f4f19d
$ git log -1 59f4a51a7f2c53b9fd161b032d0fcb8a85f4f19d
commit 59f4a51a7f2c53b9fd161b032d0fcb8a85f4f19d
Author:     Matt Caswell <matt@openssl.org>
AuthorDate: Thu Aug 26 10:03:51 2021 +0100
Commit:     Pauli <pauli@openssl.org>
CommitDate: Tue Aug 31 20:47:25 2021 +1000

    Add a test for verifying an email with a bad othername type
    
    Reviewed-by: Tomas Mraz <tomas@openssl.org>
    Reviewed-by: Paul Dale <pauli@openssl.org>
    (Merged from https://github.com/openssl/openssl/pull/16443)
````

**Comment by klssstis** [comment]
thanks for the clarification, that was helpful.
those. we can say that fixes were made on a feature that appeared two months ago and was not used in release versions.

in which case my original question is over, thanks for the reply and your time.
